Makefile

```diff
@@ -2,9 +2,19 @@ ci: clean tools deps lint
 
 deps:
 	r10k puppetfile install --verbose --moduledir modules
+	mkdir -p inspec/profiles/inspec-aem
+	mkdir -p inspec/profiles/inspec-aem-aws
+	mkdir stage
+	cd stage
+	wget https://github.com/shinesolutions/inspec-aem/archive/master.tar.gz
```

Comment:
Move inspec profiles downloading to deps target, and use inspec `vendor` command for consistency with aem-test-suite https://github.com/shinesolutions/aem-test-suite/blob/master/Makefile#L10 .